packages/client/src/components/Post.js

```diff
@@ -1,69 +1,102 @@
-import { useState, useEffect } from "react";
-import axios from "axios";
-import mockPost from "./mockPost";
+import { useState } from "react";
```

Comment:
We can simplify this a lot by using the components we've already created.

Can we instead abstract https://github.com/100devs/course-tracker/blob/main/packages/client/src/CreatePost.js to a component and go from there?

If not that abstracting the shared components between this and [CreatePost](https://github.com/100devs/course-tracker/blob/main/packages/client/src/CreatePost.js).